packages/crypto/src/lib/crypto.ts

```diff
@@ -359,14 +358,44 @@ function base64ToBufferAsync(base64) {
     });
 }
 
-async function getAmdCert(url: string) {
-  // unfortunately, until AMD enables CORS, we have to use a proxy when in the browser
-  // This project is hosted on heroku and uses this codebase: https://github.com/LIT-Protocol/cors-proxy-amd
-  if (isBrowser()) {
-    // CORS proxy url
-    url = `https://cors.litgateway.com/${url}`;
+/**
+ * Asynchronously fetches an AMD certification from a specified URL using a CORS proxy.
+ * This approach is used to bypass CORS restrictions when accessing external APIs that do not support CORS.
+ * It attempts to fetch the AMD cert through a proxy and if the proxy fetch fails, it throws an error.
+ * 
+ * @param url The URL from which to fetch the AMD cert.
+ * @returns A Promise that resolves to a Uint8Array containing the AMD certification data.
+ * @throws An error detailing HTTP or network issues encountered during the fetch process.
+ */
+async function getAmdCert(url: string): Promise<Uint8Array> {
+
+  const proxyUrl = `${LIT_CORS_PROXY}/${url}`;
+
+  console.log(`Fetching AMD cert using proxy URL due to CORS restrictions.`);
```

Comment:
should we use `log` instead of `console`
also i think we should log out the `url` from the attestation report so users may audit the cert which was fetched through the proxy